src/orca/services/nextflowtower/client.py

```diff
@@ -134,7 +136,7 @@ def get(self, path: str, **kwargs) -> dict[str, Any]:
             A dictionary from deserializing the JSON response.
         """
         json = self.request_json("GET", path, **kwargs)
```

Comment:
I think there is room for improvement here (And correct me if I am wrong). The way I understand this is currently working:

1. `request_json` is called to get the first set (Or all results if the count is low enough)
2. If there are enough results you need to drop to your paginated result set
3. You get the first set again (You already got it in #1), and then use the offset param to get all subsequent results.

Ideally you should retain the items you originally got and start at that offset in the `request_paged` call

Reply:
@BryanFauble and I had a quick huddle to discuss this, and then after toying around with some small changes I think if we are going to implement a change to make this more efficient it will need to be a separate ticket due to its potential to impact so much of the `nextflowtower` service (out of scope for this ticket)